modules/react/text/stories/BodyText.stories.mdx

```diff
@@ -0,0 +1,36 @@
+import {Meta} from '@storybook/addon-docs';
+
+import {Basic} from './examples/BodyText/Basic';
+
+import {TypeLevelComponent} from './PropTables.splitprops.tsx';
+
+<Meta title="Components/Text/Body Text/React" />
+
+# Canvas Kit Body Text
+
+`BodyText` provides a simple way to render body-level text with built-in support for Canvas type
+tokens.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-react
+```
+
+## Usage
+
+### Basic Example
+
+`BodyText` is a Type Level component (along with [`Title`](/components/text/title/),
+[`Heading`](/components/text/heading/), and [`Subtext`](/components/text/subtext/)) built on top of
+[`Text`](/components/text/text/) which provides a simple way to render body-level text according to
+the [Canvas type hierarchy](/tokens/type/#type-styles).
```

Comment:
This link will not work properly in Storybook (the `#type-styles` doesn't correspond to an `id` on the destination page in SB), but it should work properly on the Canvas Site. There are a handful of other links in this PR which will have the same issue in SB.